docs/install.rst

```diff
@@ -304,11 +304,20 @@ section of ``mexopts.sh``.
 3) Windows: tips for compiling
 -------------------------------   
    
-We have users who have adjusted the makefile to work with Windows. Please
-try::
+We have users who have adjusted the makefile to work - at least to some extent - on Windows 10. Please make sure to have a recent version of Mingw at hand, preferably with a 64bit version of gnu-make like the WinLibs standalone build of GCC and MinGW-w64 for Windows. Note that most MinGW-w64 distributions, such as TDM-GCC, do not feature the 64bit gnu-make. Fortunately, this limitation is only relevant to run the tests. To build the static and dynamic library try::
 
-  cp make.inc.windows_mingw make.inc
-  make test -j
+  copy make.inc.windows_mingw make.inc
+  make lib CPATH=(PATH_OF_FFTW_H) LIBRARY_PATH=(PATH_OF_FFTW(F)_DLL)
```

Comment:
Since CPATH etc appear 3 times here, I think better to tell user to edit make.inc and add vars there.

Reply:
Unfortunately, it is not possible to simply assign them within the make.inc file. However, I implemented a workaround which does not require additional changes of the makefile itself.